### PR TITLE
fix(curriculum): extra word in step 47 cafe menu project

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3cade99dda4e6071a85dfd.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-47
 
 # --description--
 
-You will come back to the styling the menu in a few steps, but for now, go ahead add a second `section` element below the first for displaying the desserts offered by the cafe.
+You will come back to styling the menu in a few steps, but for now, go ahead add a second `section` element below the first for displaying the desserts offered by the cafe.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3cade99dda4e6071a85dfd.md:10]:
```suggestion
You will come back to styling the menu in a few steps, but for now, go ahead and add a second `section` element below the first for displaying the desserts offered by the cafe.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
One more grammar fix while we're here 😁

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Review by lynxlynxlynx — APPROVED** [review]

**Comment by lynxlynxlynx** [comment]
Thanks and congratulations on your first accepted submission!

**Pull request merged**